src/System.CommandLine/Parsing/RootCommandResult.cs

```diff
@@ -11,6 +11,13 @@ internal class RootCommandResult : CommandResult
         private readonly Dictionary<ICommand, CommandResult> _allCommandResults;
         private readonly Dictionary<IOption, OptionResult> _allOptionResults;
 
+        internal new static readonly RootCommandResult Empty = new RootCommandResult(
```

Comment:
I would inline this. It's an implementation detail of `ParseResult.Empty` and so no change should be needed here.